plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMergeSink.java

```diff
@@ -408,7 +411,7 @@ private Slice writeDeletionVector(
 
         DeletionVectorEntry deletionVectorEntry;
         try {
-            deletionVectorEntry = writeDeletionVectors(fileSystem, rootTableLocation, deletedRows);
+            deletionVectorEntry = writeDeletionVectors(fileSystem, rootTableLocation, deletedRows, randomPrefixLength);
```

Comment:
Why is this specific to deletion vector files ? It feels like it should apply to all files to reduce chances of throttling

Reply:
I was wondering the same thing when investigating Spark behavior, but that is their current implementation:

https://github.com/delta-io/delta/blob/cb352c24809097021ad0b2811e419a2e01841cde/spark/src/main/scala/org/apache/spark/sql/delta/DeltaConfig.scala#L514-L519
↓
https://github.com/delta-io/delta/blob/cb352c24809097021ad0b2811e419a2e01841cde/spark/src/main/scala/org/apache/spark/sql/delta/util/Utils.scala#L41-L48
↓
https://github.com/delta-io/delta/blob/cb352c24809097021ad0b2811e419a2e01841cde/spark/src/main/scala/org/apache/spark/sql/delta/commands/DMLWithDeletionVectorsHelper.scala#L350-L359

Reply:
FWIW I remember seeing this approach for data files in Databricks as well.

Reply:
Yeah, we have some static test resources which have random prefix Databricks generated for data files.

Reply:
Then we should consider doing that as a follow-up, it seems strange to only do this for delete files